### DISCUSSION TITLE
ER Diagram save location change

### DISCUSSION THREAD

**Discussion by aviennas** [dbeaver/dbeaver#35001] [Q&A] [answered]
Hello, how can the save location for ER Diagrams change? It currently picks a certain project but I want to be able to choose a different one. Is this possible?

**Comment by E1izabeth** [comment, marked as answer]
DBeaver Creates new diagram in a default project.
1. You can switch the default project to create a diagram in another one with `Set active project` button.
![image](https://github.com/user-attachments/assets/38c0ea15-a596-46d0-a272-59b2fa9546cc)

2. Or you can Save it in another place, if you press `Save diagram as ...` button and choose `*.erd` format.
![image](https://github.com/user-attachments/assets/578435e4-18eb-46bd-972c-ad538fe2de31)
Then you can put the diagram to the Diagrams folder of the needed project.
![image](https://github.com/user-attachments/assets/386a9762-fe67-4744-811b-93a3e56f7852)
However, diagram is connected to the datasource, which you might not have in other project. So, when you open that diagram you will be asked to choose datasource
![image](https://github.com/user-attachments/assets/1742b13d-7058-4705-a9ed-bedc2304069e)

**Reply by aviennas** [reply]
Thank you very much! Excellent!!!
